tests/x11regressions/libreoffice/libreoffice_recent_documents.pm

```diff
@@ -36,8 +36,13 @@ sub run {
     wait_still_screen;
     x11_start_program('oowriter');
     send_key "alt-f";
+    # Because of bsc#1074057 alt-f is not working in libreoffice under wayland
+    # use another way to replace alt-f in SLED15
+    if (sle_version_at_least('15')) {
```

Comment:
Same as above for `is_sle` :)

Reply:
Thanks, I just fixed it, please help to review.